### DISCUSSION TITLE
How to solve the OSError `No lab.conf in given directory`?

### DISCUSSION THREAD

**Discussion by Lix01786** [Kidel/Kathara#251] [Q&A] [answered]
Hello, i was trying to start a lab and got this error message:
![image](https://github.com/KatharaFramework/Kathara/assets/150289836/43b3e0aa-3745-45bb-8117-ffa0b91d970e)
even though i checked in the lab folder and the lab.conf is there. How can i fix it?

Thanks in advance

**Comment by tcaiazzi** [comment, marked as answer]
Hello @Lix01786,

The error says that the `kathara lstart` command did not find a `lab.conf` file inside the current directory. 

Probably, you are running the command on the wrong directory. Ensure to enter the network scenario directory before running the `kathara lstart` command.

Please let me know if this helps to solve the error (and, in case, mark this as answer 😇). 

Cheers, 
Tommaso
